fury/tests/test_utils.py

```diff
@@ -256,5 +259,14 @@ def test_rotate(interactive=False):
         npt.assert_equal(red_sum_new > red_sum, True)
 
 
+def test_matplotlib_figure_to_numpy():
+    fig = matplotlib.pyplot.figure()
+    matplotlib.pyplot.plot([1, 2, 3])
+    matplotlib.pyplot.close()
+    arr = matplotlib_figure_to_numpy(fig)
+    test_arr = np.zeros_like(arr)
+    npt.assert_raises(AssertionError, npt.assert_array_equal, arr, test_arr)
```

Comment:
I do not understand this. Can you explain me?

Reply:
Do the comments help? Is there some better way to test this? Newbie at testing viz.. Any advice?